tests/impl.cpp

```diff
@@ -469,6 +469,27 @@ result_t test_mm_andnot_ps(const SSE2NEONTestImpl &impl, uint32_t i)
     return r;
 }
 
+// r0 := ~a0 & b0
+// r1 := ~a1 & b1
+result_t test_mm_andnot_pd(const SSE2NEONTestImpl &impl, uint32_t i)
+{
+    result_t r = TEST_FAIL;
+    const double *_a = (const double *) impl.mTestFloatPointer1;
+    const double *_b = (const double *) impl.mTestFloatPointer2;
+
+    __m128d a = _mm_load_pd(_a);
+    __m128d b = _mm_load_pd(_b);
+    __m128d c = _mm_andnot_pd(a, b);
+    // now for the assertion...
```

Comment:
The commit message should be more detailed.
As other developers might not be aware that the assertion here means the assertion in `validateXXXX`.

I think you should describe why you do the bitwise operation on `uint64_t` instead of `double` since bitwise operation does not accept `float/double` data type.